cpp/installer.cc

```diff
@@ -323,6 +323,9 @@ void CreateVuFindDatabases(const VuFindSystemType vufind_system_type, DbConnecti
         }
     }
 
+    std::string error__;
```

Comment:
- Variable names ending with underscores are reserved for class members only
- This is written to but never read